openmetadata-airflow-apis/openmetadata_managed_apis/api/routes/ip.py

```diff
@@ -40,7 +63,23 @@ def get_host_ip():
     """
 
     try:
-        return ApiResponse.success({"ip": requests.get("https://api.ipify.org").text})
+
+        # Try to pick up IP from ipify
+        host_ip = _get_ip_safely(IPIFY_URL)
+
+        # If that service is not available, let's use my-ip as a fallback
+        if not host_ip:
+            host_ip = _get_ip_safely(MY_API_URL)
```

Comment:
nit: loop in a list in case we want to add a third option or have even more

Reply:
oh yeah agree, thanks